metaci/release/models.py

```diff
@@ -61,7 +61,7 @@ class Release(StatusModel):
         _("version name"), max_length=255, null=True, blank=True
     )
     version_number = models.CharField(
-        _("version number"), max_length=255, null=True, blank=True
+        _("version number"), max_length=255, null=True, blank=False
```

Comment:
We should change both `null` and `blank` to False. A field with null=True and blank=False can still be set to null in code, which we don't want.